recipes/utf8proc/all/conanfile.py

```diff
@@ -4,17 +4,17 @@
 from conan.tools.microsoft import is_msvc
 import os
 
-required_conan_version = ">=1.47.0"
+required_conan_version = ">=1.53.0"
 
 
 class Utf8ProcConan(ConanFile):
     name = "utf8proc"
+    description = "A clean C library for processing UTF-8 Unicode data"
+    license = "MIT expat"
```

Comment:
```suggestion
    license = "MIT"
```

Same license and looks like https://github.com/JuliaStrings/utf8proc/blob/master/LICENSE.md is right, but for some reaosn this used the other naming - onñy MIT is valid spdx

Reply:
@RubenRBS 
Thank you!